src/aws_s3_config_bucket.tsx

```diff
@@ -37,11 +60,25 @@ export class BucketConfigurator extends React.Component<Props>{
         const bucketName = target.bucketName.value;
 
         // Reuse the S3 object from App
-        const s3Auth = this.props.existingS3Obj!;
+        const s3 = this.props.existingS3Obj!;
```

Comment:
This might look to similar to the actual AWS S3 function, but otherwise it's good.